kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go

```diff
@@ -106,7 +106,7 @@ type Kubernetes struct {
 }
 
 type Kurl struct {
-	AdditionalNoProxyAddresses   []string `json:"additionalNoProxyAddresses,omitempty" yaml:"proxyAddress,omitempty"`
```

Comment:
was this intended?

Reply:
Isn't that a bug. It's the wrong yaml tag. Or am I misunderstanding your comment?

Reply:
I misread the +- 🤦